gpytorch/lazy/kronecker_product_lazy_tensor.py

```diff
@@ -188,7 +188,7 @@ def _symeig(self, eigenvectors=True):
             evals_, evecs_ = eval_tensor.double().symeig(eigenvectors=eigenvectors)
 
             # we chop any negative eigenvalues
-            evals_ = evals_.clamp_min(0.0)
+            evals_.clamp_min_(0.0)
```

Comment:
I think we should still avoid doing in-place tensor updates, as there could be other more general use cases in which we'd want to backprop through `symeig`. The additional mem allocation is dwarfed by the actual computations anyway, so there is no efficiency issue.

Reply:
At the moment as coded it will never backprop through symeig. If a PR changes this behavior then this should probably not be in place, yeah. Can change this now to avoid a gotcha, I suppose

Reply:
I'll put up a quick PR